chaoslib/openebs/fetch_data_from_replica.yml

```diff
@@ -0,0 +1,65 @@
+- name: Get replica id
+  shell: kubectl get pods {{ rpod }} -n openebs -o jsonpath='{.spec.containers[*].env[0].value}'
+  register: cstor_id
+
+- name: Check for dataset name
+  shell: kubectl exec -it {{ rpod }} -n  openebs --container cstor-pool -- zfs list cstor-{{ cstor_id.stdout }}/{{ pv.stdout }}
+  register: ds_exists
+
+#- name: delete existing snapshot
+#  shell: kubectl exec -it {{ rpod }} -n  openebs --container cstor-pool -- zfs destroy cstor-{{ cstor_id.stdout }}/app-percona-ns-percona-mysql-claim-4001043235@verify
+#register: snap_destroy
+
+- name: Take snapshot on dataset
```

Comment:
taking snapshot at istgt will be better so that we can be sure that there is no difference in the data, even though this is used a util from any other litmus tests

Reply:
ok